src/allmydata/test/cli/wormholetesting.py

```diff
@@ -29,17 +29,21 @@ async def local_peerB(helper, mw):
     import wormhole
     run(peerA(wormhole))
 """
+# mypy: warn-unused-configs, disallow-any-generics, disallow-subclassing-any, disallow-untyped-calls, disallow-untyped-defs, disallow-incomplete-defs, check-untyped-defs, disallow-untyped-decorators, warn-redundant-casts, warn-unused-ignores, warn-return-any, no-implicit-reexport, strict-equality, strict-concatenate
+# This inline mypy config applies a per-file mypy config for this file.
+# It applies the '--strict' list of flags to this file.
+# If you want to test using CLI run the command remove the inline config above and run:
+# "mypy --follow-imports silent --strict src/allmydata/test/cli/wormholetesting.py"
```

Comment:
Yes I know inline for per-file configurations does not seem like it is a great idea. I have tried the implementation and the strict flag configuration will be applied to the whole codebase. 
```
[mypy]
ignore_missing_imports = True
plugins=mypy_zope:plugin
show_column_numbers = True
pretty = True
show_error_codes = True
warn_unused_configs =True
no_implicit_optional = True
warn_redundant_casts = True
strict_equality = True

[mypy-allmydata.test.cli.wormholetesting]
warn_unused_configs = True
disallow_any_generics = True
disallow_subclassing_any = True
disallow_untyped_calls = True
disallow_untyped_defs = True
disallow_incomplete_defs = True
check_untyped_defs = True
disallow_untyped_decorators = True
warn_redundant_casts = True
warn_unused_ignores = True
warn_return_any = True
no_implicit_reexport = True
strict_equality = True
strict_concatenate = True
```
I would this be a better compromise?

Reply:
Does it work?  I tried something like that and found it generated 20,000+ new errors across the whole codebase.

Reply:
If I use the ```strict``` flag I get 20,000+ errors but with the above implementation I do not get any. 

It also works as intended for me if I run it on master. It has the same results as running ```mypy -strict src/allmydata/test/cli/wormholetesting.py```  both with 6 errors.

Reply:
Okay.  If that works consistently then I think that mypy.ini is a better approach, especially since you can also have a comma-separated list of module names all sharing a single configuration block and also play tricks with wildcards to get different modules to share one config block (supposedly).